README.md

```diff
@@ -2142,6 +2143,81 @@ Happy New Moon with Report is an open-source implementation of WebAssembly writt
     </tr>
     </table>
 
+
+
+## <a name="wazero"></a>[wazero](https://github.com/tetratelabs/wazero) <sup>[top⇈](#contents)</sup>
+> wazero is a WebAssembly 1.0 spec compliant runtime written in Go, with zero platform dependencies.
+
+* **Languages written in**
+
+    <table>
+    <tr>
+        <td>Go</td>
+    </tr>
+    </table>
+
+* **Compiler framework**
+
+    <table>
+    <tr>
+        <td>Custom</td>
+    </tr>
+    </table>
+
+
+* **Compilation / Execution modes**
+
+    <table>
+    <tr>
+        <td>Interpreted</td>
+        <td>JIT</td>
+    </tr>
+    </table>
+
+* **Interoperability with other languages**
+
+    - `N/A`
+
+* **Non-MVP features supported**
+
+    <table>
+    <tr>
+        <td>Import/Export of Mutable Globals</td>
+        <td>Sign-extension operators</td>
+        <td>Multi-Value Returns</td>
+    </tr>
+    </table>
+
+* **Host APIs supported**
+
+    <table>
+    <tr>
+        <td>WASI</td>
+    </tr>
+    </table>
+
+* **Non-web standards**
+
+    <table>
+    <tr>
+        <td>WASI</td>
+    </tr>
+    </table>
+
+* **Used by**
+
+    - `N/A`
+
+* **Platforms supported**
+
+    <table>
+    <tr>
+        <td>Linux</td>
```

Comment:
PS we also test "scratch" as we are the only option that allows a static binary built in Go to do that. Ex 3ish MB Docker image inclusive of the normal Go 1.5ish baseline https://github.com/tetratelabs/wazero#platform

Also, users tend to be interested in both platform and arch dimensions. Most alternatives skip windows or skip windows on one arch. wazero is tested for JIT on amd64 and arm64 as well interpreter adds testing for riscv64*

* since the interpreter uses what go does, it may work on other archs, it is just we happen to test it on riscv64

Reply:
Yeah, wasn't sure which platforms to include on there, so went with what everything else included, even though technically more should be possible. For now I've included the JIT supported archs, and a note that the interpreter should work anywhere that Go can compile to.

Reply:
thanks for the help!